### ISSUE TITLE
AuditLog not initialized before repository connector itself is initialized

### ISSUE THREAD

**Issue by cmgrote** [odpi/egeria#2368]
At least when configuring a connection using the repository proxy service, the AuditLog of the (remote) repository connector is not initialized before the repository connector itself is initialized.  The result is that we have no access to record anything into the audit log as part of the (remote) repository connector initialization.

(Conversely, for the (remote) event mapper connection the audit log _does_ appear to be initialized before the initialization of the event mapper itself.)

**Comment by mandy-chessell** [comment]
@cmgrote this is correct - the audit log is not guaranteed to be available until start().  Initialize should be used just to save values - should not do anythhing until start()

**Comment by cmgrote** [comment]
Ok, thanks -- will move this to my repo as a bug in my connectors 😬
